tools/rna_tools/reago/reago.xml

```diff
@@ -0,0 +1,140 @@
+<tool id="reago" name="Reago" version="0.1.0">
+    <description> to assemble rRNA</description>
+
+    <requirements>
+        <requirement type="package" version="1.1">infernal</requirement>
+        <requirement type="package" version="1.5.7">genometools</requirement>
+        <requirement type="package" version="1.1">reago</requirement>
+    </requirements>
+
+    <stdio>
+        <exit_code range="1:" />
+    </stdio>
+
+    <command>
+<![CDATA[
+        #set $r1_output=$os.path.splitext($os.path.basename(str($r1_file)))[0]
+        #set $r2_output=$os.path.splitext($os.path.basename(str($r2_file)))[0]
+
+        python $__tool_directory__/format_reago_input_files.py 
+            --r1_input "$r1_file"
+            --r2_input "$r2_file"
+            --r1_output $r1_output
+            --r2_output $r2_output
+
+        ; 
```

Comment:
please combine commands only with && and not with ';'.
This is because ';' will proceed with the following command even if the previous command failed.